lib/opentok/opentok.rb

```diff
@@ -148,7 +148,7 @@ def initialize(api_key, api_secret, opts={})
     def create_session(opts={})
 
       # normalize opts so all keys are symbols and only include valid_opts
-      valid_opts = [ :media_mode, :location, :archive_mode ]
+      valid_opts = [ :media_mode, :location, :archive_mode, :e2ee ]
```

Comment:
Please add docs to the comment above this (on line 146):

```
    # @option opts [Symbol] : e2ee Determines whether to enable
    # {https://tokbox.com/developer/guides/end-to-end-encryption end-to-end encryption}
    # for the session.

````

Reply:
Thanks @jeffswartz. This is still a draft PR at this stage. I'll add the docs comments and also update the README before finalizing. I'm also planning to add some validation on the opts when creating a session, but am awaiting further info on this (my assumption is that `e2ee` should **not** be set to `true` if `archive_mode` is `always`, or if `media_mode` is relayed).

Will rev version and update CHANGES if this is going out as a stand-alone release. If it gets batched together in a release with the Audio Connector feature, I'll do that separately to cover both features.